x-pack/plugins/ingest_manager/server/services/settings.ts

```diff
@@ -41,10 +49,11 @@ export async function saveSettings(
     };
   } catch (e) {
     if (e.isBoom && e.output.statusCode === 404) {
-      const res = await soClient.create<SettingsSOAttributes>(
-        GLOBAL_SETTINGS_SAVED_OBJECT_TYPE,
-        newData
-      );
+      const defaultSettings = createDefaultSettings();
```

Comment:
@nchaulet since during setup we create the Settings saved object with default attributes, I moved those into a function and they will be added when the saved object (if missing for some reason) is created here as well.